apps/store/src/features/priceCalculator/DeductibleSelectorV2.tsx

```diff
@@ -64,16 +62,30 @@ export function DeductibleSelectorV2({ offers, selectedOffer, onValueChange }: P
           </CardRadioGroup.Item>
         ))}
       </CardRadioGroup.Root>
-      <Link
-        href={PageLink.deductibleHelp({ locale })}
-        style={{ alignSelf: 'center' }}
-        target="_blank"
-        rel="noopener"
-      >
-        <Text as="span" size="xs">
-          {t('DEDUCTIBLE_SELECTOR_FOOTER_LINK')}
-        </Text>
-      </Link>
+      <DeductibleInfo />
     </>
   )
 }
+
+function DeductibleInfo() {
+  const { t } = useTranslation('purchase-form')
+  const deductibleInfo = usePriceCalculatorDeductibleInfo()
+  return (
+    <FullscreenDialog.Root>
+      <FullscreenDialog.Trigger asChild={true}>
+        <Button variant="secondary" size="small" className={sprinkles({ alignSelf: 'center' })}>
+          {t('DEDUCTIBLE_SELECTOR_FOOTER_LINK')}
+          <PlusIcon size="1rem" />
+        </Button>
+      </FullscreenDialog.Trigger>
+      <FullscreenDialog.Modal>
+        <FullscreenDialog.Title className={sprinkles({ display: 'none' })}>
```

Comment:
Once my PR is merged you can use the global visuallyHidden class instead
```suggestion
        <FullscreenDialog.Title className="visuallyHidden">
```